src/modules/map/components/cesium/CesiumWMSLayer.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+    <div>
+        <slot />
+    </div>
+</template>
+
+<script>
+import { CoordinateSystem, WEBMERCATOR } from '@/utils/coordinateSystems'
+import { ImageryLayer, Rectangle, WebMapServiceImageryProvider } from 'cesium'
+import { WGS84_EXTENT } from '@/config'
+import { mapState } from 'vuex'
+import { YEAR_TO_DESCRIBE_ALL_OR_CURRENT_DATA } from '@/api/layers/LayerTimeConfigEntry.class'
+import GeoAdminWMSLayer from '@/api/layers/GeoAdminWMSLayer.class'
+import addImageryLayerMixins from '@/modules/map/components/cesium/utils/addImageryLayer-mixins'
+
+// todo should we adapt LOD to resolution?
+const MAXIMUM_LEVEL_OF_DETAILS = 18
```

Comment:
What is the concept of LOD in Cesium, so that I understand what we could achieve by implementing what you suggest?

Reply:
from Cesium docs:
> The maximum level-of-detail supported by the imagery provider, or undefined if there is no limit. If not specified, there is no limit.

In old geo.admin (https://github.com/geoadmin/mf-geoadmin3/blob/c21a537ea0f449279a7b715599d1ba64f601f718/src/components/map/LayersService.js#L424) maxResolution takes from layer config and then corresponding value takes from an array of LODs or 18 if not exists.

Should we implement something similar?